src/FileCabinet/SuiteScripts/HC_importInventoryAdjustment.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @NApiVersion 2.1
+ * @NScriptType ScheduledScript
+ */
+define(['N/sftp', 'N/task'], function (sftp, task) {
+
+    function execute(context) {
+      // Establish a connection to a remote FTP server
+      /* The host key can be obtained using OpenSSH's ssh-keyscan tool:
+          ssh-keyscan -t <hostKeyType> -p <port> <hostDomain>
+          Example: ssh-keyscan -t ECDSA -p 235 hc-uat.hotwax.io 
+      */
+      
+      var hostKey = 'AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBKNOKFqtVrDd8YvJ8c82fBL4/KK9R69iYJAckMMKrLggmmnzQWGc5h8CVx3vsIHEw3WoDe10eWUQtRRhv+gE2CQ=';
```

Comment:
Do not commit key on git repo, also please delete this commit from history. 
do rebase

Reply:
Removed the host key.